dev/cloud/naming_conventions.md

```
@@ -0,0 +1,259 @@
+workshop_order.status
+resource.tractor_unit
+```
+
+
+## Web API 
+
+### Request 
+#### Method Usage
+
+##### GET
+|                                            |     |     
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)